src/styles/style.scss

```diff
@@ -1,5 +1,6 @@
 @import "atoms/button";
 @import "atoms/collapsible";
+@import "colors";
```

Comment:
Import colors first, then you don't need to import them in the atoms. Otherwise you got a redundancy.